logify.js

```diff
@@ -0,0 +1,51 @@
+const isPlainObject = require('lodash/isPlainObject');
+const { logger: _logger } = require('./index');
+
+function formatValue(arg) {
+  if (typeof arg === 'function') {
+    return `ƒ() ${arg.name} {...}`;
+  }
+  if (arg && typeof arg === 'object' && !isPlainObject(arg)) {
+    return arg.constructor.name;
+  }
+  return JSON.stringify(arg) || '';
+}
+
+function formatArguments(args) {
+  const fullArgs = args.map(formatValue).join(', ');
+  return fullArgs.length > 100 ? `${fullArgs.slice(0, 100)}...` : fullArgs;
+}
+
+function formatMethod(method, args) {
+  return `${method}(${formatArguments(args)})`;
+}
+
+module.exports = function logify(target, property, desc, level = 'verbose') {
```

Comment:
so you can use either `@logify` or `@logify('info')`?

Reply:
ya